sql/src/test/resources/calcite/tests/window/shuffleOperators.sqlTest

```diff
@@ -0,0 +1,72 @@
+type: "operatorValidation"
+
+sql: |
+    SELECT
+      countryName,
+      cityName,
+      __time,
+      ROW_NUMBER() OVER (PARTITION BY countryName ORDER BY cityName),
+      ROW_NUMBER() OVER (ORDER BY countryName desc, cityName),
+      ROW_NUMBER() OVER (ORDER BY countryName, cityName),
+      ROW_NUMBER() OVER (PARTITION BY cityName),
+      ROW_NUMBER() OVER ()
+    FROM wikipedia
+    where page < '0' and channel like '#en%'
+
+expectedOperators:
+  - type: "naivePartition"
+    partitionColumns: [ ]
+  - type: "window"
+    processor:
+      type: "rowNumber"
+      outputColumn: "w4"
+  - type: "naiveSort"
+    columns:
+      - column: "cityName"
+        direction: "ASC"
+  - type: "naivePartition"
+    partitionColumns: [ "cityName" ]
+  - type: "window"
+    processor:
+      type: "rowNumber"
+      outputColumn: "w3"
+  - type: "naiveSort"
+    columns:
+      - column: "countryName"
+        direction: "ASC"
+      - column: "cityName"
+        direction: "ASC"
+  - type: "naivePartition"
+    partitionColumns: [ ]
```

Comment:
is this empty `partitionColumns` expected?

Reply:
yes it is expected as w2 is corresponding to `(ORDER BY countryName, cityName)` and the prior partitionColumns is not empty.